node/typings/Search.ts

```diff
@@ -139,6 +139,7 @@ interface BiggySearchProduct {
   textAttributes: BiggyTextAttribute[]
   split: BiggySplit
   categoryTrees: BiggyCategoryTree[]
+  clusterHighlights: Record<string, string>
```

Comment:
I think it would be better to create a type for ClusterHighlight, that way we make it clearer what a cluster highlight is and the props it has.

Reply:
I used the same type as the solr uses, it was easier, do you think that should I keep this way?

Reply:
I think the best option would be to create a type. but it's just a suggestion, you can keep it this way